spark-and-cassandra-on-centos/azuredeploy.json

```diff
@@ -0,0 +1,807 @@
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Premium_LRS"
+      ]
+    },
+    "vmMasterVMSize": {
+      "metadata": {
+        "description": "VM size for master Spark node.  This VM can be sized smaller."
+      },
+      "type": "string",
+      "defaultValue": "Standard_D1_v2",
+      "allowedValues": [
+        "Standard_D1_v2",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D5_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2",
+        "Standard_A8",
+        "Standard_A9",
+        "Standard_A10",
+        "Standard_A11"
+      ]
+    },
+    "vmNumberOfSlaves": {
+      "metadata": {
+        "description": "Number of VMs to create to support the slaves.  Each slave is created on it's own VM.  Minimum of 2 & Maximum of 200 VMs."
+      },
+      "type": "int",
+      "minValue": 2,
+      "maxValue": 200
+    },
+    "vmSlaveVMSize": {
+      "metadata": {
+        "description": "VM size for slave Spark nodes.  This VM should be sized based on workloads."
+      },
+      "type": "string",
+      "defaultValue": "Standard_D3_v2",
+      "allowedValues": [
+        "Standard_D1_v2",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D5_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2",
+        "Standard_A8",
+        "Standard_A9",
+        "Standard_A10",
+        "Standard_A11"
+      ]
+    },
+    "vmCassandraVMSize": {
+      "metadata": {
+        "description": "VM size for Cassandra node.  This VM should be sized based on workloads."
+      },
+      "type": "string",
+      "defaultValue": "Standard_D3_v2",
+      "allowedValues": [
+        "Standard_D1_v2",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D5_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2",
+        "Standard_A8",
+        "Standard_A9",
+        "Standard_A10",
+        "Standard_A11"
+      ]
+    },
+    "vmAdminUserName": {
+      "metadata": {
+        "description": "Specific an admin username that should be used to login to the VM."
+      },
+      "type": "string",
+      "minLength": 1
+    },
+    "vmAdminPassword": {
+      "metadata": {
+        "description": "Specific an admin password that should be used to login to the VM."
+      },
+      "type": "securestring"
+    },
+    "_artifactsLocation": {
+      "metadata": {
+        "description": "Auto-generated container in staging storage account to receive post-build staging folder upload"
+      },
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/spark-and-cassandra-on-centos/CustomScripts"
+    },
+    "_artifactsLocationSasToken": {
```

Comment:
Still need the Sas Token in the event that we want to use Azure Storage Blobs to keep the artifacts.